### ISSUE TITLE
Sonoff US Model of 3-button T1

### ISSUE THREAD

**Issue by cirolaferrara** [arendst/Sonoff-Tasmota#2824]
Flashed without problem with mirko/SonOTA five Sonoff US Model of 3-button T1 but after firmware upgrade, i bricked one item.
Can you help me to repair it? No known method to put it into flash mode? Thanks

**Comment by curzon01** [comment]
[Sonoff-T1 Wiki](https://github.com/arendst/Sonoff-Tasmota/wiki/Sonoff-T1-UK-and-T1-EU) does not help?

**Comment by cirolaferrara** [comment]
It says:

```
US Model of 3-button T1, received direct from Sonoff China 2018-05-12
Front photo (Google photos) Rear photo (Google photos) (Sorry about the tape smudges).

The header labels on J3 are accurate (apparently some boards have TX and RX reversed; this doesn't seem to suffer that).

Flashed with SONOTA.exe. Above method of getting into flash mode doesn't seem to work.
```

**Comment by curzon01** [comment]
I read it too

**Comment by cirolaferrara** [comment]
ehmm how?

**Comment by curzon01** [comment]
Does your board looks same like on the photos on [US Model of 3-button T1, received direct from Sonoff China 2018-05-12](https://github.com/arendst/Sonoff-Tasmota/wiki/Sonoff-T1-UK-and-T1-EU#us-model-of-3-button-t1-received-direct-from-sonoff-china-2018-05-12). 

Unfortunately a) I don't have a T1 US and b) the photos are to bad to find something.

Do it yourself: Search for the ESP chip and then search for the chip layout (use Google) and find GPIO 0.Thas all what you need.

If you don't find all of it, do you have detailed photos from the front and rear of your device?

**Comment by cirolaferrara** [comment]
These are photos:
https://pasteboard.co/HmVXARu.jpg
https://pasteboard.co/HmVXUVx.jpg
https://pasteboard.co/HmVYD5C.jpg

**Comment by curzon01** [comment]
Have a look on U4 on your board - that's the ESP.

And the ESP Pinout:
![esp8266ex](https://user-images.githubusercontent.com/6636844/40574297-8d9ff162-60cf-11e8-9020-b834cbddd2a9.jpg)

So using the dot * on upper left corner (same for your U4 chip) the GPIO0 should be here:

![t1 us gpio0](https://user-images.githubusercontent.com/6636844/40574293-764f38c4-60cf-11e8-866b-85fb9659a0f2.png)

**Comment by cirolaferrara** [comment]
Doh. I need a very good solder. Mine is very bad to do this work

**Comment by curzon01** [comment]
Yeah, that's my origin question - it does not work with an wood painting solder :grinning:
May be you have an ohmmeter, follow the circuit path and find a better junction...

**Comment by cirolaferrara** [comment]
Done! (I hope so). And now?

**Comment by curzon01** [comment]
Connect pin GPIO 0 to GND during (only necessary during power up, not steady) power up to switch device into flash mode, then flash it
